### DISCUSSION TITLE
Add publisher facet to REST API response

### DISCUSSION THREAD

**Discussion by amandafrench** [datacite/datacite-suggestions#136] [General Suggestions]
### What is the problem that your suggestion solves?

I'm interested in seeing the top values in the `publisher` element in DataCite metadata, and I can't easily see that without doing a search on a specific string

### What solution might meet your needs?

Add the `publisher` element to the list of metadata facets returned by an API query along with providers, clients, affiliations etc. 
<img width="432" alt="Screenshot 2024-11-19 at 5 08 32 PM" src="https://github.com/user-attachments/assets/0ecccfe5-933f-4530-ac34-e1d4196221f5">

### Your name

Amanda French

### Your organization

Crossref

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by codycooperross** [comment]
Thanks, @amandafrench ! This is helpful. What workflows would a publisher facet help you with? Noting that we also introduced a non-required [publisherIdentifier field in Schema 4.5](https://datacite-metadata-schema.readthedocs.io/en/4.5/properties/publisher/#a-publisheridentifier) that might also be useful field for faceting in the future.

**Reply by amandafrench** [reply]
Thanks, @codycooperross! I admit that the workflow I'd find it useful for might not be a common one, but we are tracking the uptake of ROR IDs in DataCite records at https://ror.org/about/#impact, and we always find it interesting to use the facets to see which clients and providers are the top providers of ROR IDs for affiliations, funders, and publishers. My sense is that the publisher might often be the same as the client or provider (as for instance with NIFS), but it's hard to tell. Would specifically be interested in doing a query filtered by items with the type "journal article" and seeing publisher values for those in the facets, and/or as you say the values for the publisher identifiers, which would normally be the ROR IDs.

Hope that helps!
